src/søknad/steg/4-barnasbosted/OmAndreForelder.tsx

```diff
@@ -225,16 +231,11 @@ const OmAndreForelder: React.FC<Props> = ({
           />
         </KomponentGruppe>
       )}
-      {forelder.hvorforIkkeOppgi?.svarid === EHvorforIkkeOppgi.annet && (
+      {forelder.hvorforIkkeOppgi?.verdi === EHvorforIkkeOppgi.Annet && (
```

Comment:
Vil dette bli riktig? verdi er jo språkavhengig, men svarId er enums

Reply:
Gode innspill, sjekker nå for 'Annet' og 'Other'